docs/development.md

```diff
@@ -67,9 +66,13 @@ db information.
 
 ```
 cd gateway-ha/target/
-java -jar gateway-ha-{{VERSION}}-jar-with-dependencies.jar server ../gateway-ha-config.yml
+java --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED -jar gateway-ha-{{VERSION}}-jar-with-dependencies.jar server ../gateway-ha-config.yml
 ```
 
+If you encounter `module java.base does not "opens java.lang" to unnamed module` error, 
+this may be due to newer version of java.
+Do check opts at [jvm.config](../.mvn/jvm.config) when running java command.
```

Comment:
Not sure what this means ..

Reply:
Oh what i meant was that the guide might be out-of-date so if any error happens with `module java.base does not "opens java.lang" to unnamed module` error, user can check .mvn/jvm.config file for other jvm opts. I noticed that [trinodb/trino](https://github.com/trinodb/trino/blob/master/.mvn/jvm.config) has more opts.